mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -1031,12 +1096,40 @@ func flattenUpdatePolicy(updatePolicy *compute.InstanceGroupManagerUpdatePolicy)
 	return results
 }
 
+<% unless version == "ga" -%>
+func expandAllInstancesConfig(configured []interface{}) *compute.InstanceGroupManagerAllInstancesConfig {
+	allInstancesConfig := &compute.InstanceGroupManagerAllInstancesConfig{}
+	for _, raw := range configured {
+		data := raw.(map[string]interface{})
+		properties := &compute.InstancePropertiesPatch{}
+    properties.Metadata = convertStringMap(data["metadata"].(map[string]interface{}))
+    properties.Labels = convertStringMap(data["labels"].(map[string]interface{}))
+		allInstancesConfig.Properties = properties
+	}
+	return allInstancesConfig
+}
+
+func flattenAllInstancesConfig(allInstancesConfig *compute.InstanceGroupManagerAllInstancesConfig) []map[string]interface{} {
+  results := []map[string]interface{}{}
+  if allInstancesConfig != nil {
+    props := map[string]interface{}{}
+    props["metadata"] = allInstancesConfig.Properties.Metadata
+    props["labels"] = allInstancesConfig.Properties.Labels
+    results = append(results, props)
+  }
+  return results
+}
+<% end -%>
+
 func flattenStatus(status *compute.InstanceGroupManagerStatus) []map[string]interface{} {
 	results := []map[string]interface{}{}
 	data := map[string]interface{}{
 		"is_stable":      status.IsStable,
 		"stateful":       flattenStatusStateful(status.Stateful),
 		"version_target": flattenStatusVersionTarget(status.VersionTarget),
+    <% unless version == "ga" -%>
+		"all_instances_config": flattenStatusAllInstancesConfig(status.AllInstancesConfig),
+		<% end -%>
```

Comment:
We should only set this field conditionally on whether or not  `allInstancesConfig` is set.

I suspect we we are failing due to indexing into this and failing.